### PR TITLE
Changing Thimble URL to Glitch URL 

### LINKED ISSUE #8101: Content bug: The link to Thimble service.  RU version
## What page(s) did you find the problem on?

Russian version of page:
https://developer.mozilla.org/ru/docs/Learn/JavaScript/First_steps/Silly_story_generator

## Specific page section or heading?

```
<div class="note notecard" id="sect3">
<p><strong>Замечание</strong>:  Так же вы можете использовать такие сайты как  <a class="external external-icon" href="http://jsbin.com/" rel=" noopener">JSBin</a> или **<a class="external external-icon" href="https://thimble.mozilla.org/" rel=" noopener">Thimble</a>** для выполнения вашего испытания. Вы можете вставить HTML, CSS и JavaScript в один из этих онлайн редакторов. Если онлайн редактор, который вы используете, не имеет отдельного окна для JavaScript - не стесняйтесь вставить все скрипты в <code>&lt;script&gt;</code> элемент внутри  HTML страницы.</p>
</div>
```

## What is the problem?

the link doesnt work  `<a class="external external-icon" href="https://thimble.mozilla.org/" rel=" noopener">Thimble</a>
`
## What did you expect to see?

the Thimble moved to Glitch.io, need to update link

### PR DESCRIPTION
**Pull request by rohini-ranjanR**
### Description
The Thimble URL in https://developer.mozilla.org/ru/docs/Learn/JavaScript/First_steps/Silly_story_generator is not working mentioned in issue #8101. I linked the link to Glitch code editor page to this page https://glitch.com/edit/#!/steady-phase-recorder?path=README.md%3A1%3A0

Change Thimble code editor URL to Glitch Code editor URL as Thimble moved to Glitch mentioned in issue #8101
### Related issues and pull requests
"fixes #8101"

### CODE REVIEW

**Comment by Africanj1952** [comment]
Markdownlint (PR files)

**Comment by rohini-ranjanR** [comment]
Thank you **@Africanj1952**

File: files/ru/learn/javascript/first_steps/silly_story_generator/index.md [hunk 1]
```diff
@@ -32,7 +32,9 @@ original_slug: Learn/JavaScript/Первые_шаги/Создатель_глу
...
 - Перейти на [страницу, содержащую исходный текст](https://github.com/mdn/learning-area/blob/master/javascript/introduction-to-js-1/assessment-start/raw-text.txt), и оставить её открытой в отдельной вкладке браузера. Она вам понадобится позже.
 
-> **Примечание:** **Замечание**: Так же вы можете использовать такие сайты как [JSBin](http://jsbin.com/) или [Thimble](https://thimble.mozilla.org/) для выполнения вашего испытания. Вы можете вставить HTML, CSS и JavaScript в один из этих онлайн редакторов. Если онлайн редактор, который вы используете, не имеет отдельного окна для JavaScript - не стесняйтесь вставить все скрипты в `<script>` элемент внутри HTML страницы.
+> **Примечание**
+
+**Замечание**: Так же вы можете использовать такие сайты как [JSBin](http://jsbin.com/) или [Glitch](https://glitch.com/edit/) для выполнения вашего испытания. Вы можете вставить HTML, CSS и JavaScript в один из этих онлайн редакторов. Если онлайн редактор, который вы используете, не имеет отдельного окна для JavaScript - не стесняйтесь вставить все скрипты в `<script>` элемент внутри HTML страницы.
```
Comment (sashasushko) [inline files/ru/learn/javascript/first_steps/silly_story_generator/index.md:37]:
```suggestion
> **Примечание:** Так же вы можете использовать такие сайты как [JSBin](http://jsbin.com/) или [Glitch](https://glitch.com/edit/) для выполнения вашего испытания. Вы можете вставить HTML, CSS и JavaScript в один из этих онлайн-редакторов. Если онлайн-редактор, который вы используете, не имеет отдельного окна для JavaScript – не стесняйтесь вставить все скрипты в `<script>`-элемент внутри HTML-страницы.
```

**Review by sashasushko — APPROVED** [review]

**Comment by sashasushko** [comment]
@rohini-ranjanR спасибо 🙏

**Pull request merged**